src/cmd/linuxkit/run_qemu.go

```diff
@@ -375,10 +378,12 @@ func runQemuContainer(config QemuConfig) error {
 	var args []string
 	config, args = buildQemuCmdline(config)
 
-	// if user specify the "-fw" parameter, this should override the default in container context,
-	// with "-v" option, we will have the chance to assign an external FW binary to the containerized qemu
-	// instead of the fixed FW bin instealled by the build process of the image.
-	if config.UEFI {
+	// If we are running in a container and if the the user
+	// specifies the "-fw" parameter to a non-default, then we
+	// assume that the FW binary is on the local filesystem and
+	// bind mount it into the container. Otherwise, the FW image
+	// in the container is used.
```

Comment:
I don't really want to default to `""` as this is a bit tedious for users. I would think that using the systems default `/usr/share/ovmf/bios.bin` is far less common. Or would you object?